<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>CryoTheraPy</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Jobs and Set-Up</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_makeJobTabs">
          <property name="text">
           <string>Set Jobs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_in_paths">
          <property name="title">
           <string>Paths and Set-Up</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_in_paths">
             <item>
              <widget class="QLabel" name="lbl_path_movies">
               <property name="text">
                <string>Path to movies directory:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="line_path_movies"/>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_path_mdoc">
               <property name="text">
                <string>Path to mdoc file directory:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="line_path_mdoc"/>
             </item>
             <item>
              <widget class="QPushButton" name="btn_use_movie_path">
               <property name="text">
                <string>Same as path to movies</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_config">
               <property name="text">
                <string>Microscope used</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="dropDown_config"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Start Relion</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>150</y>
          <width>141</width>
          <height>191</height>
         </rect>
        </property>
        <property name="title">
         <string>Path to new Project</string>
        </property>
        <widget class="QPushButton" name="btn_writeStar">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>140</y>
           <width>61</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Write Star</string>
         </property>
        </widget>
        <widget class="QWidget" name="formLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>121</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="line_path_new_project"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="btn_changeStarValues">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>370</y>
          <width>100</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Commit Changes</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_python2">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>400</y>
          <width>131</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Switch to Python 2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_start">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>430</y>
          <width>80</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
